mdp4e.py

```diff
@@ -517,38 +514,3 @@ def pomdp_value_iteration(pomdp, epsilon=0.1):
                 },
     }
 """
-
-
-# __________________________________________________________________________
-# Chapter 17 Multiagent Planning
-
-
-def double_tennis_problem():
```

Comment:
Where did this function go?

Reply:
This planning problem is already defined in planning.py. It was a duplicate code!